apps/manager/src/hooks/use-blockscout-api.ts

```diff
@@ -24,8 +32,10 @@ export function addressApiEndpoint(chainId: number) {
 
 export function useTokenBalances({
   address,
+  chain,
 }: {
   address?: Address.Address
+  chain?: 'base'
```

Comment:
Let's add `undefined`. Should we use `chainId` instead of name?

```suggestion
  chain?: 'base' | undefined
```